.github/actions/VerifyAppChanges/action.yaml

```diff
@@ -0,0 +1,13 @@
+name: Verify App Changes
```

Comment:
Sounds a little bit broad. This string will show up in the PR so it might be a good idea to be more specific about what exactly this check does. E.g. "Test Preprocessor Symbols" or "Verify Preprocessor Symbols"